Core/Core/Pages/PageList/PageListViewController.storyboard

```diff
@@ -73,15 +73,25 @@
                                                         <userDefinedRuntimeAttribute type="string" keyPath="iconName" value="arrowOpenRightSolid"/>
                                                     </userDefinedRuntimeAttributes>
                                                 </imageView>
+                                                <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="7bY-hM-aJf">
+                                                    <rect key="frame" x="0.0" y="0.0" width="382" height="76"/>
+                                                    <connections>
+                                                        <action selector="frontPageTapped:" destination="JGx-7A-Apd" eventType="touchUpInside" id="ajK-L4-YhU"/>
```

Comment:
Should we use `primaryActionTriggered` instead of `touchUpInside`?

Reply:
I can switch it.  What would be the pros of doing this?

Reply:
I'm not positive. I'm pretty sure VO will send both. But it does sound more semantic, right?

Reply:
I prefer `primaryActionTriggered`. If we use it everywhere it saves us from having to remember what the primary action is for the different UIControls.

Reply:
You mean `valueChanged` and `touchUpInside`?